### ISSUE TITLE
Uncaught TypeError: Cannot read property 'trim' of undefined

### ISSUE THREAD

**Issue by starsy** [eugeny-dementev/winston-console-formatter#8]
```
Uncaught TypeError: Cannot read property 'trim' of undefined
      at node_modules/winston-console-formatter/dist/utils.js:60:61
      at Array.forEach (<anonymous>)
      at arrayProperty (node_modules/winston-console-formatter/dist/utils.js:59:10)
      at typifiedString (node_modules/winston-console-formatter/dist/utils.js:69:14)
      at node_modules/winston-console-formatter/dist/utils.js:36:40
      at Array.forEach (<anonymous>)
      at objectProperty (node_modules/winston-console-formatter/dist/utils.js:34:20)
      at typifiedString (node_modules/winston-console-formatter/dist/utils.js:71:14)
      at node_modules/winston-console-formatter/dist/utils.js:36:40
      at Array.forEach (<anonymous>)
      at objectProperty (node_modules/winston-console-formatter/dist/utils.js:34:20)
      at typifiedString (node_modules/winston-console-formatter/dist/utils.js:71:14)
      at node_modules/winston-console-formatter/dist/utils.js:36:40
      at Array.forEach (<anonymous>)
      at objectProperty (node_modules/winston-console-formatter/dist/utils.js:34:20)
      at Object.metaToYAML (node_modules/winston-console-formatter/dist/utils.js:118:10)
      at Object.formatter (node_modules/winston-console-formatter/dist/index.js:33:31)
      at Object.exports.log (node_modules/winston/lib/winston/common.js:222:27)
      at exports.Console.Console.log (node_modules/winston/lib/winston/transports/console.js:99:19)
      at transportLog (node_modules/winston/lib/winston/logger.js:234:15)
      at node_modules/winston/node_modules/async/lib/async.js:157:13
      at _each (node_modules/winston/node_modules/async/lib/async.js:57:9)
      at Object.async.each (node_modules/winston/node_modules/async/lib/async.js:156:9)
      at exports.Logger.Logger.log (node_modules/winston/lib/winston/logger.js:246:9)
      at exports.Logger.target.(anonymous function) [as debug] (node_modules/winston/lib/winston/common.js:54:18)
```

**Comment by starsy** [comment]
PR for fix: https://github.com/eugeny-dementev/winston-console-formatter/pull/7

**Comment by eugeny-dementev** [comment]
Fixed in `1.0.0-beta.0`
